Assurance Cases.md

```diff
@@ -75,14 +75,18 @@ As for Home Assitant official add-ons, we presume that they are secure and if a
 
-.red[TODO] Insert Evidence 5
+As discussed in R1, a concerning risk of any out of network communication is a Man in the Middle Attack. This is a type of attack in which a user stands between two communicating devices, and attempts to hijack data mid-transfer. However to combat this Home Assistant ensures that all traffic between the remote device and local network are encrypted, using [Let's Encrypt SSL certificates](https://www.nabucasa.com/config/remote/#how-it-works), ensuring that data can only be read by the local network. 
+
+As for R2, we analze the case in which an attacker obtains the secure certificate by other means. However, as explained in sub-claim C3 it is required that once a connection is established between two machines the local Home Assistant credentials are required in order to allow for [authorization](https://www.nabucasa.com/config/remote/#security). It also a known feature, that Home Assistant allows tracking of all events done on both remote and local sessions allowing user [customized guard locks to be put in place](https://www.home-assistant.io/docs/authentication/). However, the storage of local credentials in Home Assistant is its own caveat that is covered in [Assurance Case 1](#assurance-case-1).
+
+As discussed in R3, another possibility is that an attacker is able to generate their own key and attach it to your local Home Assistant instance. However as explained in sub-claim C4, connection certificates are only generated at setup time by Home Assistant or are manually configured on [network](https://www.nabucasa.com/config/remote/#remote-ui). This does bring one glaring worry, however, in the case of Home Assistant manually decrypting traffic while routing it from your remote device, to your Home Assistant integration. This scenario is publically announced, however, and Home Assistant verifies that in no normal circumstance will they read the data in transit. The only situation in which they will decrypt user data is when being forced by a government body, as outline in Undermine UM2 and discussed in the official  [Home Assistant Documentation](https://www.nabucasa.com/config/remote/#security) 
```

Comment:
The beginning of the last sentence reads a little weird ... This scenario is publically announced, however, and Home Assistant ...